docs/modules/ROOT/pages/auth/auth-directive.adoc

```diff
@@ -74,6 +74,9 @@ The full list of operations and how they related to Cypher clauses are:
 
 |`DISCONNECT`
 |`MATCH` and `DELETE`
+
+|`SUBSCRIBE`
+| `<NONE>`
```

Comment:
I wonder if this table really has any value at all - do our users really need to know how our rules map to Cypher? Especially given our library is meant to be an abstraction over Cypher.

Reply:
I don't think is valuable tbh. If you agree I'll remove the whole table